sphinx_a4doc/diagram_directive.py

```diff
@@ -108,6 +108,14 @@ def visit_node_html(self: sphinx.writers.html.HTMLTranslator, node):
             self.body.append(svg)
             self.body.append('</p>')
 
+    @staticmethod
+    def visit_node_text(self: sphinx.writers.text.TextTranslator, node):
+        if node['options'].alt:
+            self.add_text('{}'.format(node['options'].alt))
+        else:
+            self.add_text(yaml.dump(node['diagram']))
```

Comment:
Call for proposal. If there is any better way to produce pretty output, please drop your idea.

Reply:
Can't think of any better way. We could generate diagrams using ASCII graphics, but I'm not sure it's worth the effort.